examples/ionoscloud/compute/serverset.yaml

```diff
@@ -96,10 +98,14 @@ spec:
         image: "28d0fa34-927f-11ee-8008-6202af74e858"
         size: 10
         type: HDD
-#        todo - add sshKeys or imagePassword
-#        todo - add userdata in volume creation
         userData: "" #cloud-config
         imagePassword: "thisshouldbesecret"
+        substitutions:
+          - additionalProperties:
+              cidr: "10.0.0.0/24"
+            key: ipv4Address
+            type: ipv4Address
```

Comment:
hey filip,

`type` is the `type` of the substitution, whereas `key` is the `key` to be replaced by `type` within the incoming document.

Reply:
By incoming document, you mean the raw  cloud-init config string?

Reply:
yes, so it's up to the user to set the proper `key` - we are just replacing

Reply:
Ok, we can go with it like this too. I only wanted to say that from the documentation perspective, It would have been a good idea for the person who is looking at this example to know that key does not need to be the same as the type so that it doesn't create confusion.